python/tank/deploy/tank_commands/setup_project_wizard.py

```diff
@@ -641,4 +641,4 @@ def execute(self):
             core_localize.do_localize(self._log, 
                                       self._params.get_configuration_location(sys.platform), 
                                       suppress_prompts=True,
-                                      strip_toolkit_credentials=self._is_session_based_authentication_supported())
```

Comment:
Sorry Rob, I had to merge it in to unblock Jack who was testing the Desktop app. We can remove the dead code later if you feel like having the option in the admin ui is not a good idea.